src/messages.rs

```diff
@@ -610,17 +610,14 @@ pub enum MessageContent {
     SectionSplit(Prefix<XorName>, XorName),
     /// Sent amongst members of a newly-merged section to allow synchronisation of their routing
     /// tables before notifying other connected peers of the merge.
-    OwnSectionMerge {
-        sender_prefix: Prefix<XorName>,
-        merge_prefix: Prefix<XorName>,
-        sections: SectionMap,
-    },
+    ///
+    /// Source authority is the `sender_prefix` and destination authority is the `merge_prefix`.
```

Comment:
I'd prefer if this explained that it's send from one of the merging sections to the `PrefixSection` including both. Let's not refer to fields that don't exist anymore (`merge_prefix`).